data_steward/validation/ehr_union.py

```diff
@@ -918,6 +919,7 @@ def create_load_aou_death(bq_client, project_id, input_dataset_id,
 def main(input_dataset_id,
          output_dataset_id,
          project_id,
+         run_as,
```

Comment:
We cannot add impersonation here. This is run by app engine which cannot use impersonation